dist/filter-menu/ds4/filter-menu.css

```diff
@@ -0,0 +1,154 @@
+span.filter-menu-aria,
```

Comment:
We need to cater for `div` version too. Usually the only difference between the span and div versions is that span is inline-block.

Reply:
Yeah, let me clean that up a little, as that's too specific.